libpod/container_inspect.go

```diff
@@ -506,6 +506,15 @@ func (c *Container) generateInspectContainerHostConfig(ctrSpec *spec.Spec, named
 
 	// Annotations
 	if ctrSpec.Annotations != nil {
+		if len(ctrSpec.Annotations) != 0 {
+			if hostConfig.Annotations == nil {
+				hostConfig.Annotations = make(map[string]string)
+			}
+			for k, v := range ctrSpec.Annotations {
+				hostConfig.Annotations[k] = v
+			}
+		}
```

Comment:
this can be just `hostConfig.Annotations = ctrSpec.Annotations`

Reply:
Done